modules/python/types/source/types.i

```diff
@@ -47,3 +47,93 @@
 
 %template(VectorRowColInt) std::vector<types::RowCol<sys::SSize_T> >;
 %template(VectorSizeT) std::vector<size_t>;
+
+// Pickle utilities
+%pythoncode
+%{
+    import cPickle as pickle
+%}
+
+%extend types::RowCol<double>
```

Comment:
Good question. I had looked into this originally and found that one can easily extend the c++ templates, but didn't come up with a way to wrap the python code. I'll put a little more thought into it and let you know what I learn.